main.go

```diff
@@ -180,14 +180,16 @@ func runGRPCServer(ctx context.Context, deps *gRPCServerDeps) {
 	managementpb.RegisterSecurityChecksServer(gRPCServer, managementgrpc.NewChecksServer(checksSvc))
 
 	// TODO remove PERCONA_TEST_IA once IA is out of beta: https://jira.percona.com/browse/PMM-7001
-	if enable, _ := strconv.ParseBool(os.Getenv("PERCONA_TEST_IA")); enable {
+	if enable, err := strconv.ParseBool(os.Getenv("PERCONA_TEST_IA")); err == nil && enable {
 		l.Warnf("Enabling experimental IA APIs.")
 		iav1beta1.RegisterAlertsServer(gRPCServer, ia.NewAlertsService())
 		iav1beta1.RegisterChannelsServer(gRPCServer, ia.NewChannelsService())
 		iav1beta1.RegisterRulesServer(gRPCServer, ia.NewRulesService())
 		iav1beta1.RegisterTemplatesServer(gRPCServer, ia.NewTemplatesService())
 	}
 
+	// FIXME Checking settings.DBaaS.Enabled there is not correct: that setting can be changed via API,
```

Comment:
we can't change it via API at the moment, so it's better to update the text that we should fix it with ability to update DBaaS.Enabled via API.

Reply:
Indeed